### ISSUE TITLE
Bug: Exception in Visual Studio 2017

### ISSUE THREAD

**Issue by AlexBroitman** [KhaosCoders/VSCodeBlockEndTag#19]
Here is the error I got (from ActivityLog.xml):
```
  <entry>
    <record>1005</record>
    <time>2019/02/11 15:21:58.636</time>
    <type>Error</type>
    <source>Editor or Editor Extension</source>
    <description>System.NullReferenceException: Object reference not set to an instance of an object.&#x000D;&#x000A;   
at Microsoft.CodeAnalysis.Editor.Implementation.TextStructureNavigation.AbstractTextStructureNavigatorProvider.TextStructureNavigator.TryFindLeafToken(SnapshotPoint point, SyntaxToken&amp; token, CancellationToken cancellationToken)&#x000D;&#x000A;   
at Microsoft.CodeAnalysis.Editor.Implementation.TextStructureNavigation.AbstractTextStructureNavigatorProvider.TextStructureNavigator.FindLeafNode(SnapshotSpan span, CancellationToken cancellationToken)&#x000D;&#x000A;   
at Microsoft.CodeAnalysis.Editor.Implementation.TextStructureNavigation.AbstractTextStructureNavigatorProvider.TextStructureNavigator.GetSpanOfEnclosingWorker(SnapshotSpan activeSpan, CancellationToken cancellationToken)&#x000D;&#x000A;   
at Microsoft.CodeAnalysis.Editor.Implementation.TextStructureNavigation.AbstractTextStructureNavigatorProvider.TextStructureNavigator.&lt;&gt;c__DisplayClass9_1.&lt;GetSpanOfEnclosing&gt;b__0(IWaitContext waitContext)&#x000D;&#x000A;   
at Microsoft.VisualStudio.LanguageServices.Implementation.Utilities.VisualStudioWaitIndicator.Wait(String title, String message, Boolean allowCancel, Boolean showProgress, Action`1 action)&#x000D;&#x000A;   
at Microsoft.CodeAnalysis.Editor.Implementation.TextStructureNavigation.AbstractTextStructureNavigatorProvider.TextStructureNavigator.GetSpanOfEnclosing(SnapshotSpan activeSpan)&#x000D;&#x000A;   
at CodeBlockEndTag.CBETagger.GetTagsCore(SnapshotSpan span)&#x000D;&#x000A;   at CodeBlockEndTag.CBETagger.GetTags(SnapshotSpan span)&#x000D;&#x000A;   
at CodeBlockEndTag.CBETagger.&lt;Microsoft-VisualStudio-Text-Tagging-ITagger&lt;Microsoft-VisualStudio-Text-Editor-IntraTextAdornmentTag&gt;-GetTags&gt;d__21.MoveNext()&#x000D;&#x000A;   
at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.&lt;GetTagsForBuffer&gt;d__39.MoveNext()&#x000D;&#x000A;
--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   
at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)</description>
  </entry>

```

**Comment by propagating** [comment]
Same issue here.

**Comment by profcinders** [comment]
I'm getting the same thing. It happens the very first time I close a tab after opening Visual Studio.

**Comment by Khaos66** [comment]
Should be fixed in the new version 2.0.2 ;)
